python/ray/rllib/agents/ppo/appo_policy.py

```diff
@@ -134,30 +156,35 @@ def __init__(self,
                 clip_pg_rho_threshold=tf.cast(clip_pg_rho_threshold,
                                               tf.float32))
 
-        logp_ratio = tf.exp(actions_logp - prev_actions_logp)
+        self.is_ratio = tf.clip_by_value(
+            tf.exp(prev_actions_logp - old_policy_actions_logp), 0.0, 2.0)
```

Comment:
why is this here?

Reply:
Nothing changed except for format.sh reformatting the lines.